### PR TITLE
Add new sf data examples

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Pls review @richardellison - there is more coming that I will build-up here.

File: R/geo-functions.R [hunk 1]
```diff
@@ -176,13 +176,43 @@ bbox_scale <- function(bb, scale_factor){
...
+}
+
+#' @export
+geo_bb.Spatial <- function(bb) {
+  bb2poly(bb)
+}
+
+#' @export
+geo_bb.sf <- function(bb) {
+  bb <- sf::as_Spatial(sf::st_geometry(bb))
```
Comment (richardellison) [inline R/geo-functions.R]: I think this should be:

```{r}
bb <- sf:::as_Spatial(sf::st_geometry(bb))
```

**Comment by richardellison** [comment]
Seems to fail because of as_Spatial (see comment above).

File: R/geo-functions.R (hunk 1 above)
Comment (Robinlovelace) [inline R/geo-functions.R]: Yes but `as_Spatial()` should soon be exported by @edzer. See https://github.com/r-spatial/sf/commit/c2a493ed19482fdfc955c0bf9ab4a7eb133f1315 and my follow-on PR.

**Comment by Robinlovelace** [comment]
I don't think CRAN allows you to use `pkg:::fun()`: https://stackoverflow.com/questions/32535773/using-un-exported-function-from-another-r-package

**Comment by Robinlovelace** [comment]
@richardellison this is good to merge now I think now that `as_Spatial()` is exported.

**Pull request merged**